grease/src/Grease/Macaw/FunctionOverride.hs

```diff
@@ -60,8 +60,8 @@ import What4.FunctionName qualified as W4
 import What4.Protocol.Online qualified as W4
 
 -- | A Macaw function override, corresponding to a single S-expression file.
-data MacawFunctionOverride p sym arch =
-  MacawFunctionOverride
+data MacawSExpOverride p sym arch =
+  MacawSExpOverride
     { mfoPublicFnHandle :: MacawFnHandle arch
```

Comment:
All of these fields are prefixed with `mfo-`, which stands for `MacawFunctionOverride`. Now that we've renamed `MacawFunctionOverride` to `MacawSExpOverride`, we should change the prefixes to `mso-`.